src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -4,31 +4,78 @@ description: Release notes for Nuxeo Drive.
 tree_item_index: 700
 review:
   comment: 'release'
-  date: '2021-07-06'
+  date: '2021-07-29'
   status: ok
 toc: true
 ---
 
-Welcome to the Release Notes for **Nuxeo Drive 5.2.4**
+Welcome to the Release Notes for **Nuxeo Drive 5.2.5**
 
-**Status**: <font color="#0066ff">**Release**</font> </br>
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/5.2.4.md)
+**Status**: <font color="#ff0000">**Beta**</font> </br>
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/5.2.5.md)
 
 ## General
 
+### Fixes
+
+#### Restored Support for macOS 10.13
+
+The support for macOS 10.13 was accidentally lost after the Nuxeo Drive 5.2.1 release. The application build process has been reviewed and the support for this version has been restored.
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2724](https://jira.nuxeo.com/browse/NXDRIVE-2724).
+
+#### `ca-bundle` Option Ignored When No SSL Verification Done
+
+Previously, the `ca-bundle` option was set to `None` or removed when the value of `ssl-no-verify` was changed at runtime. This behaviour was bad as it could cause errors and forced the user had to manually set the option again when re-enabling the SSL verification.
```

Comment:
```suggestion
Previously, the `ca-bundle` option was set to `None` or removed when the value of `ssl-no-verify` was changed at runtime. This behaviour was bad as it could cause errors and forced the user to manually set the option again when re-enabling the SSL verification.
```